### ISSUE TITLE
Warning: Object.keys called on non-object

### ISSUE THREAD

**Issue by bierik** [sindresorhus/grunt-eslint#47]
Here is my eslint configuration:

```javascript
eslint: {
  target: ["Gruntfile.js", "test/**/*.js", "scripts/**/*.js"]
}
```

When i got this warning, the linter doesn't show the file.
When i define the target just to "Gruntfile.js" where I do not call Object.keys,  the warning appears anyway.
Setting options to an empty object was not the solution.
I tried to flag with "--verbose" and "--debugger" as well but got no errors in console.

Got version 5.1.0

**Comment by sindresorhus** [comment]
What does `grunt eslint --stack` output?

**Comment by bierik** [comment]
```
TypeError: Object.keys called on non-object
    at Function.keys (native)
    at mergeConfigs (/Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/util.js:35:12)
    at /Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/util.js:60:25
    at Array.forEach (native)
    at mergeConfigs (/Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/util.js:35:25)
    at /Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/util.js:60:25
    at Array.forEach (native)
    at Object.mergeConfigs (/Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/util.js:35:25)
    at Config.getConfig (/Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/config.js:282:19)
    at processFile (/Users/kbieri/bricoler/simplelayout/node_modules/grunt-eslint/node_modules/eslint/lib/cli-engine.js:131:31)
```

**Comment by sindresorhus** [comment]
The error is coming from ESLint. Try reinstalling this task. If you're still having the issue, open an issue on https://github.com/eslint/eslint/issues/new with the same info.
